packages/Webkul/Shop/src/Routes/customer-routes.php

```diff
@@ -141,9 +141,7 @@
                     /**
                      * Wishlist.
                      */
-                    Route::get('wishlist', [WishlistController::class, 'index'])->defaults('_config', [
-                        'view' => 'shop::customers.account.wishlist.wishlist',
-                    ])->name('shop.customer.wishlist.index');
+                    Route::get('wishlist', [WishlistController::class, 'index'])->name('shop.customer.wishlist.index');
```

Comment:
shop.customers.account.wishlist.index

Fix all the wishlist routes also. Consult with Amit.

Reply:
Done